cypress/support/selectorsSignalDetails.js

```diff
@@ -17,7 +18,7 @@ export const CHANGE_STATUS = {
   buttonClose: '[class*="incident-detail__preview-close"]',
```

Comment:
Close buttons will be targetable by `data-testid="closeButton"`

Reply:
There is no `data-testid="closeButton"` for the X button when changing status

Reply:
'will be' Targetable. Not right now, but they will after #712 has been merged. It was more like a heads-up.